js/theme/fonts.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Utility functions to manage font properties to style mapping for both iOS and Android
+ * Fonts are managed differently on Android and iOS. Read the Font section of the
+ * README file included in this repository.
+ *
+ * @flow
+ */
+
+import { Platform } from 'react-native'
+
+const font = Platform.OS === 'android' ? 'TitilliumWeb' : 'Titillium Web'
```

Comment:
by looking at the [Plaform code](https://github.com/facebook/react-native/blob/26684cf3adf4094eb6c405d345a75bf8c7c0bf88/Libraries/Utilities/Platform.ios.js) I just noticed that there's a very convenient [Platform.select](https://facebook.github.io/react-native/docs/platform-specific-code.html#platform-module) method available.

```js
const font = Platform.select({
  android: 'TitilliumWeb',
  iOS: 'Titillium Web'
})
```